src/main/java/org/sagebionetworks/web/client/widget/team/JoinTeamWidget.java

```diff
@@ -79,6 +82,8 @@ public JoinTeamWidget(JoinTeamWidgetView view,
 		this.jsonObjectAdapter = jsonObjectAdapter;
 		this.gwt = gwt;
 		this.wikiPage = wikiPage;
+		this.progressWidget = progressWidget;
```

Comment:
Looks good to me.
![image](https://cloud.githubusercontent.com/assets/8346437/9452787/19f008b0-4a6b-11e5-82ea-d49cd58e9588.png)